phpBB/adm/style/permission_mask.html

```diff
@@ -14,7 +14,7 @@
 	<!-- ENDIF -->
 // ]]>
 </script>
-<script type="text/javascript" src="style/permissions.js"></script>
+<!-- INCLUDEJS permissions.js -->
```

Comment:
Have you checked if this still works now? Some parts of the ACP might still use onclick and similar inline JS features.